spec/components/pagination_component_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe PaginationComponent, type: :component do
+  it "renders the pagination content" do
+    pagy = Pagy.new(count: 100, page: 2)
+    render_inline described_class.new(pagy:)
+
+    within "govuk-pagination__list" do
```

Comment:
This feels like a silly question, but.. why does this work? 🤔

I'm assuming `govuk-pagination__list` is a class on a HTML element. So I'd have expected it to start with a `.` to signify that it's a CSS class selector?

```suggestion
    within ".govuk-pagination__list" do
```

Reply:
I'll get this fixed in my next PR